manifests/s/sf-yuzifu/bcm_convertor/1.2.0/sf-yuzifu.bcm_convertor.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: sf-yuzifu.bcm_convertor
+PackageVersion: 1.2.0
+InstallerLocale: zh-CN
+InstallerType: nullsoft
```

```suggestion
InstallerType: portable
```
Maybe I should modify it like this?